.circleci/behat-test.sh

```diff
@@ -24,12 +24,6 @@ echo "Behat test site: $TERMINUS_SITE.$TERMINUS_ENV"
 echo "::::::::::::::::::::::::::::::::::::::::::::::::"
 echo
 
-# Exit immediately on errors
-set -ex
```

Comment:
I moved this farther down so that we don't have to jump through hoops to hide the admin password and other sensitive information.

I saw no good reason to have `-x` for verbose command logging. It can be re-added if it is needed in the future for debugging

Reply:
I am +1 on just getting rid of -x in CI in most instances. It's hard to prove that there are no disclosures of sensitive information in a build script, and -x makes it easier for these to accidentally slip in.